### PR TITLE
Exclude Transifex (.ts/.qm) and WiX (.po/.wxl) l10n files from local Git merges

### PR DESCRIPTION
**Pull request by uklotzde**
As discussed on [Zulip](https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/Translations.20on.20transifex/near/154472443)

Please verify if this is the correct way to simplify merging between different releases, e.g. from 2.1 to 2.2.

### CODE REVIEW

**Comment by uklotzde** [comment]
Anyone? I would like to re-sync our release and development branches. This PR just states and documents how to resolve the merge conflicts with translations now and for the future.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# TODO
+# Execute the following command in your local project
+# directory to enable automatic merge resolution:
```
Comment (daschuer) [inline .gitattributes]: I think we should add here some more explanations. How about that.
... to register the shell command "true" as merge driver "ours" that does just nothing during a git merge.

**Review by daschuer — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (daschuer) [reply .gitattributes]: Ping! 
This command does not enable automatic merge resolution.
It sets a shell script that is used for all merge=ours files.
In this case the script "true" dies not touch anything, just returns successful.

**Comment by daschuer** [comment]
LGTM, thank you.

**Comment by uklotzde** [comment]
I've added a note to the [Wiki page](https://mixxx.org/wiki/doku.php/internationalization)

**Pull request merged**